features/install-cli.feature

```diff
@@ -52,6 +54,14 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
+    Then the exit status should be 0
+    And the binary "docker" should be installed with version "1.12.1"
+
+    When I run `bundle exec vagrant service-manager install-cli docker --path /tmp/docker`
```

Comment:
I don't think that _/tmp/docker_ is a good choice. This won't work on Windows for example. I know we cannot run tests on Windows at, but we are not so far from it. This would just put another roadblock into our way. Also what would be responsible to clean this up, aka removing already downloaded binaries?

What exactly is the issue with using a env variable here? Where does it break? Do you have a stack trace or failing test output?